bingo/symbolic_regression/symbolic_regressor.py

```diff
@@ -61,94 +102,119 @@ def __init__(self, *, population_size=500, stack_size=32,
         self.max_evals = max_evals
         self.scale_max_evals = scale_max_evals
 
+        if evolutionary_algorithm is None:
+            evolutionary_algorithm = AgeFitnessEA
         self.evolutionary_algorithm = evolutionary_algorithm
 
         self.clo_threshold = clo_threshold
 
+        # TODO make private attribute,
+        #  as well as other attributes not defined in __init__?
         self.best_ind = None
```

Comment:
Thoughts on making best_ind, archipelago, generator, etc. private?

Reply:
maybe worth keeping them public for use cases like we just encountered with Miles